src/app/admin/backup.ts

```diff
@@ -3,34 +3,129 @@ import {
   DropboxAccessToken,
   GcsApplicationCredentials,
   LND_SCB_BACKUP_BUCKET_NAME,
+  Nextcloudurl,
+  Nextclouduser,
+  Nextcloudpassword,
 } from "@config"
 import { Storage } from "@google-cloud/storage"
-import { Dropbox } from "dropbox"
+import axios from "axios"
+import {
+  asyncRunInSpan,
+  SemanticAttributes,
+  addAttributesToCurrentSpan,
+  addEventToCurrentSpan,
+  recordExceptionInCurrentSpan,
+} from "@services/tracing"
+import { ErrorLevel } from "@domain/shared"
 
 export const uploadBackup =
   (logger: Logger) =>
   async ({ backup, pubkey }: { backup: string; pubkey: Pubkey }) => {
     logger.debug({ backup }, "updating scb on dbx")
     const filename = `${BTC_NETWORK}_lnd_scb_${pubkey}_${Date.now()}`
 
-    try {
-      const dbx = new Dropbox({ accessToken: DropboxAccessToken })
-      await dbx.filesUpload({ path: `/${filename}`, contents: backup })
-      logger.info({ backup }, "scb backed up on dbx successfully")
-    } catch (error) {
-      logger.error({ error }, "scb backup to dbx failed")
+    if (
+      !(
+        DropboxAccessToken ||
+        GcsApplicationCredentials ||
+        (Nextcloudurl && Nextclouduser && Nextcloudpassword)
+      )
+    ) {
+      const err = new Error(
+        "Missing environment variable for LND static channel backup destination.",
+      )
+      logger.error(err)
+      recordExceptionInCurrentSpan({ error: err, level: ErrorLevel.Critical })
```

Comment:
this will trigger an alert... @nicolasburtey is dropbox required if we enable other storages?

Reply:
we could have an alert if no storage is being set up